bin/platinum.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
```

Comment:
can't we use the platinum script that already exists?

Reply:
I guess we could, but it's designed to be executed interactively so would we not end up calling it multiple times with different arguments?